### PR TITLE
Explicitly use std::set.

### PR DESCRIPTION
**Pull request by wolfman2000**
This seemed simpler than min and max.

### CODE REVIEW

File: src/AttackDisplay.cpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ void AttackDisplay::Init( const PlayerState* pPlayerState )
 			attacks.insert( asAttacks[att] );
 	}
 
-	for( set<RString>::const_iterator it = attacks.begin(); it != attacks.end(); ++it )
+	for( auto it = attacks.begin(); it != attacks.end(); ++it )
```
Comment (Wallacoloo) [inline src/AttackDisplay.cpp]: If you're going to use `auto` here, you should change this to `cbegin()` and `cend()` to retain the const-correctness that the old version has.

File: src/archutils/Win32/GraphicsWindow.cpp [hunk 2]
```diff
@@ -69,10 +69,10 @@ static LRESULT CALLBACK GraphicsWindow_WndProc( HWND hWnd, UINT msg, WPARAM wPar
 			if( !g_bHasFocus )
 			{
 				RString sName = GetNewWindow();
-				static set<RString> sLostFocusTo;
+				static std::set<RString> sLostFocusTo;
 				sLostFocusTo.insert( sName );
 				RString sStr;
-				for( set<RString>::const_iterator it = sLostFocusTo.begin(); it != sLostFocusTo.end(); ++it )
+				for( auto it = sLostFocusTo.begin(); it != sLostFocusTo.end(); ++it )
```
Comment (Wallacoloo) [inline src/archutils/Win32/GraphicsWindow.cpp]: Same here - should be `cbegin()` and `cend()` if you want the derived type to remain a `const_iterator`.

**Comment by wolfman2000** [comment]
While I appreciate that tip, my understanding is that cbegin() and cend() are a part of c++14, not c++11. Can that be verified? Some of the sources I see say it's c++14.

**Comment by Wallacoloo** [comment]
http://en.cppreference.com/w/cpp/container/vector/begin and http://www.cplusplus.com/reference/vector/vector/cbegin/ both have it marked as a c++11 method. Which sources say it's c++14?

**Comment by wolfman2000** [comment]
http://en.cppreference.com/w/cpp/iterator/begin says this is c++14.

Then again, this is the nonmember version I think.

**Comment by Wallacoloo** [comment]
Yup, that's the nonmember version.

**Pull request merged**